packages/docusaurus-plugin-content-docs/src/__tests__/index.test.ts

```diff
@@ -584,6 +584,119 @@ describe('versioned website (community)', () => {
   });
 });
 
+describe('versioned website (community) with custom versions dir', () => {
+  async function loadSite() {
+    const siteDir = path.join(__dirname, '__fixtures__', 'versioned-site');
+    const context = await loadContext({siteDir});
+    const sidebarPath = path.join(siteDir, 'community_sidebars.json');
+    const routeBasePath = 'community';
+    const pluginId = 'community';
+    const options = validateOptions({
+      validate: normalizePluginOptions as Validate<Options, PluginOptions>,
+      options: {
+        id: 'community',
+        path: 'community',
+        versionedDocsPath: 'community-versions',
```

Comment:
maybe make the path more explicit? like `community_custom_versioned_docs_path`?

Having a site with multiple versionedDocsPath (default + this one) is already a bit hard to reason about, there's a chance in 1 year I get really confused by all this 🤪